src/main/java/net/starschema/clouddb/jdbc/BQConnection.java

```diff
@@ -1058,14 +1040,4 @@ public int cancelRunningQueries() {
     public Long getMaxBillingBytes() {
         return maxBillingBytes;
     }
-
-    /**
-     * Returns true if queries on this connection should use the synchronous jobs.query api to run queries.
-     *
-     * This is the default, and the old, async API is being left in as a bailout switch in case there is some use case
-     * that only it supports that a client discovers.
-     * */
-    boolean shouldUseQueryApi() {
-        return useQueryApi;
-    }
```

Comment:
I think we maybe still use this bailout switch?

Reply:
Hmm; I no results for the following in _helltool_ directory.
```
❯ rg usequeryapi
``

So @stalbot let me know that the query API is ~4 months; but it was made the default for all BigQuery connections ?
Gives me strong confidence on it as the sole query result path.